packages/create/init.ts

```diff
@@ -98,6 +107,27 @@ function mergeGitIgnore(existingGISet: Set<string>, templateGISet: Set<string>):
     return joined.replaceAll('\n#','\n\n#');
 }
 
+function updateAppConfig(configFilePath: string, appName: string): void {
+  try {
+    const content = fs.readFileSync(configFilePath, 'utf-8');
+    const configFile = YAML.parse(content) as ConfigFile;
+    // Change the app_db_name field to the sanitized appName, replacing dashes with underscores
+    let appDbName = appName.replace('-', '_');
+    if (appDbName.match(/^\d/)) {
+      appDbName = "_" + appDbName; // Append an underscore if the name starts with a digit
+    }
+    configFile.database.app_db_name = appDbName;
+    const newContent = YAML.stringify(configFile);
```

Comment:
One issue is that this removes the comments on the top of `dbos-config.yaml` which enable schema-based Intellisense.  Could we bring them back?

Reply:
Done. It preserves comments at the top now.